src/workloads/docs/CrudActorFSMAdvanced.yml

```diff
@@ -0,0 +1,180 @@
+        WriteOperations:
+        - WriteCommand: insertOne
+          Document: {phoneID: 1, state: "Off"}
+        - WriteCommand: insertOne
+          Document: {phoneID: 2, state: "Off"}
+        - WriteCommand: insertOne
+          Document: {phoneID: 3, state: "Off"}
+        - WriteCommand: insertOne
+          Document: {phoneID: 4, state: "Off"}
+        - WriteCommand: insertOne
+          Document: {phoneID: 5, state: "Off"}
+        - WriteCommand: insertOne
+          Document: {stat: "OnOff", count: 0}
+        - WriteCommand: insertOne
+          Document: {stat: "OnSleep", count: 0}
+        - WriteCommand: insertOne
+          Document: {stat: "OnError", count: 0}
+    - &nop {Nop: true}
+    - *nop
+
+- Name: StateSmartPhoneDemo
+  Type: Crud
+  Threads: 5
+  Database: StateTest
+  Phases:
+  - *nop
+  - Duration: 30 seconds
+    Collection: CrudDemo
+    States:
+    - Name: "On"  # The device is on
+
+      # The states support the keywords Repeat, SleepBefore, SleepAfter, and Duration. These
+      # keywords behave the same as in Phase Control, but operate per state (i.e., per actor
+      # instance as it goes through the state machine.
+      Repeat: 1
+      Operations:
+      # A state can have (optional) a set of operations that execute upon entering the state. This
+      # state updates a specific document to indicate it is in state On.
+
+      # Note: Following the analogy to phases -- phases may be blocking or non-blocking. States are
+      # always non-blocking. When the phase ends, the state machine will finish any open operations,
+      # and then exit. It if is waiting for a delay, it will exit immediately. This is required as
+      # there may be states with long delays.
+      - OperationName: updateOne
+        OperationCommand:
+          Filter: &filter {phoneID: {^ActorId: {}}}
+          Update:
+            $set: {state: "On"}
+
+      # Transitions specifies next states, delays, and operations. The On state can transition to
+      # each of the three other states
+      Transitions:
+      - To: "Off"  # Transition to this state
+
+        # Transitions are probabilistically chosen based on weight. A transition with weight 2 is
+        # twice as likely to be picked as a transition with weight 1. Weights must be greater than
+        # 0.
+        Weight: 1
+
+        # Transitions also support the phase keywords of sleepBefore, sleepAfter, and repeat.
+        # ^TimeSpec is a proposed generator to randomly generate acceptable TimeSpecs with units. It
+        # accepts any valid int or double generator or constant for value.
+        SleepBefore: {^TimeSpec: {value: {^RandomDouble: {min: 0, max: 15}}, unit: "seconds"}}
+
+        # TODO: REmove the transition operations
+        # In this advanced case, operations may also be associated with transitions.
+        Operations:
+        - OperationName: updateOne
+          OperationCommand:
+            Filter: {stat: "OnOff"}
```

Comment:
`state` (couple others too)

Reply:
This actually is a "stat". I'll update to statistic.